core/src/main/java/org/apache/iceberg/PartitionsTable.java

```diff
@@ -47,7 +50,27 @@ public class PartitionsTable extends BaseMetadataTable {
             Types.NestedField.required(
                 2, "record_count", Types.LongType.get(), "Count of records in data files"),
             Types.NestedField.required(
-                3, "file_count", Types.IntegerType.get(), "Count of data files"));
+                3, "file_count", Types.IntegerType.get(), "Count of data files"),
+            Types.NestedField.required(
+                5,
+                "pos_delete_record_count",
```

Comment:
I think 'position_delete', 'equality_delete' may be better in the full form, to match SnapshotSummary.  Maybe even 'position_delete_count' , 'equality_delete_count' (SnapshotSummary has added_position_deletes, added_equality_deletes)

Reply:
fixed.